ui/index.d.ts

```diff
@@ -1251,13 +1251,20 @@ export function createAzExtOutputChannel(name: string, extensionPrefix: string):
  */
 export function openReadOnlyJson(node: { label: string, fullId: string }, data: {}): Promise<void>;
 
+export class ReadOnlyContent {
+    public readonly uri: Uri;
```

Comment:
I would prefer not to expose `uri` outside of this package, so that we're free to change how it's structured without worrying about breaking anything